name_data/labeled/company_labeled.xml

```diff
@@ -1421,4 +1421,57 @@
+  <Name><CorporationName>St.</CorporationName> <CorporationName>Frances</CorporationName> <CorporationNameOrganization>Methodist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Spring</CorporationName> <CorporationName>Creek</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Abyssinian</CorporationName> <CorporationNameOrganization>Baptist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Urban</CorporationName> <CorporationName>Grace</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Temple</CorporationName> <CorporationName>B'Nai</CorporationName> <CorporationName>Abraham</CorporationName></Name>
+  <Name><CorporationName>Madison</CorporationName> <CorporationName>Avenue</CorporationName> <CorporationNameOrganization>Baptist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>St.</CorporationName> <CorporationName>Joseph</CorporationName> <CorporationNameOrganization>Basilica</CorporationNameOrganization></Name>
+  <Name><CorporationName>Trinity</CorporationName> <CorporationName>Evangelical</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Jubilee</CorporationName> <CorporationName>Christian</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Saint</CorporationName> <CorporationName>Vincent</CorporationName> <CorporationName>de</CorporationName> <CorporationName>Paul</CorporationName></Name>
+  <Name><CorporationName>Duke</CorporationName> <CorporationNameOrganization>Chapel</CorporationNameOrganization></Name>
+  <Name><CorporationName>St.</CorporationName> <CorporationName>Augustine</CorporationName> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Beulah</CorporationName> <CorporationNameOrganization>Baptist</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Hopeful</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
+  <Name><CorporationName>Salem</CorporationName> <CorporationName>Evangelical</CorporationName> <CorporationNameOrganization>Lutheran</CorporationNameOrganization> <CorporationNameOrganization>Church</CorporationNameOrganization></Name>
```

Comment:
Similar question about `Evangelical` here.

Reply:
I agree. I will change it.